amf-client/shared/src/test/resources/validations/reports/model/invalid-reference-defined-in-lib.report

```diff
@@ -0,0 +1,38 @@
+Model: file://amf-client/shared/src/test/resources/validations/invalid-reference/error-in-lib.raml
+Profile: RAML 1.0
+Conforms? false
+Number of results: 4
```

Comment:
Why is there a new test?

Reply:
This was included when moving verifyValidReferenceKind from the document plugin interface to the raml plugin, i found a specific case that was not being tested so i included this test case (using invalid references within in a library).